### DISCUSSION TITLE
Github pages not redirecting

### DISCUSSION THREAD

**Discussion by hethwiQ** [community/community#106940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

my github pages website is not redirecting to my other github pages website and vice versa. need help with this
look at this post for more.
[https://www.reddit.com/r/github/comments/1amt725/github_pages_not_redirecting/?utm_source=share&utm_medium=web2x&context=3](url)

**Comment by queenofcorgis** [comment]
Hi there, 

If you could share the links to your website, repository, and any other relevant links you may be more likely get a response assisting you, thank you!

**Reply by hethwiQ** [reply]
sorry for forgetting, 
here's the website link [https://studenthethwiq.github.io/Work/index.html](url)
click on the **gemini redesign** project on **phone**. it doesn't seem to work.
here's the repository link [https://github.com/StudentHethwiQ/Work](url)
here's the repository link for my project page [https://github.com/StudentHethwiQ/Work](url)

please help me find the problem.

https://github.com/community/community/assets/80631564/c4e1019a-a5e2-40a0-bd94-25230dd98975
